addons/binding/org.openhab.binding.co2signal/pom.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please use tabs in XML files.

Reply:
Fixed but wondering why this isn't enforced in other bindings. I have yet to find a binding where all xml files are formatted the right way